tools/text_processing/text_processing/find_and_replace.xml

```diff
@@ -42,6 +43,9 @@
         <param name="is_regex" type="boolean" checked="false" truevalue="-r" falsevalue=""
             label="Find-Pattern is a regular expression" help="see help section for details." />
 
+        <param name="global" type="boolean" checked="true" truevalue="-g" falsevalue=""
```

Comment:
can we make it false to preserve the old default behavior? Otherwise, we can bump the build number.

Reply:
done .. .